src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/CreateClassifierTestCases.java

```diff
@@ -0,0 +1,48 @@
+package org.mule.modules.watsonalchemylanguage.automation.functional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+
+import org.junit.Test;
+import org.mule.tools.devkit.ctf.junit.AbstractTestCase;
+
+import com.admios.connector.watsonvisualrecognition.WatsonVisualRecognitionConnector;
+import com.admios.connector.watsonvisualrecognition.exceptions.VisualRecognitionException;
+import com.ibm.watson.developer_cloud.visual_recognition.v3.model.VisualClassifier;
+
+public class CreateClassifierTestCases extends AbstractTestCase<WatsonVisualRecognitionConnector> {
+
+	public CreateClassifierTestCases() {
+		super(WatsonVisualRecognitionConnector.class);
+	}
+
+	@Test(expected=IllegalArgumentException.class)
+	public void testWithNullFile() {
+		
+		try {
+			getConnector().createClassifier(null, "test-class", "Test Classifier", null);
+		} catch (VisualRecognitionException e) {
+			fail(e.getMessage());
```

Comment:
ahhhh is because the expected exception is different from what is catch, the other option is to mark the method as "throws VisualRecognitionException"... any suggestion